source/dotnet/Library/AdaptiveCards/AdaptiveTypedBaseElementConverter.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AdaptiveCards
+{
+    public abstract class AdaptiveTypedBaseElementConverter : JsonConverter
```

Comment:
Late but adding comments anyways; Good to have some class comments explainng why this wasnt needed for eg. Also do you need System.Threading.Tasks here?

Reply:
sure
and good catch! will remove it

---
In reply to: [341278327](https://github.com/microsoft/AdaptiveCards/pull/3548#discussion_r341278327) [](ancestors = 341278327)